pkg/config/profile.go

```diff
@@ -68,6 +73,10 @@ func (p *Profile) GetDeviceName() (string, error) {
 
 // GetAPIKey will return the existing key for the given profile
 func (p *Profile) GetAPIKey(livemode bool) (string, error) {
+	if os.Getenv("STRIPE_API_KEY") != "" {
+		return os.Getenv("STRIPE_API_KEY"), nil
```

Comment:
We might want to validate the API key like we do for `p.APIKey` just below.

Reply:
Good call, will fix real quick!